internal/handlers/handler_verify.go

```diff
@@ -155,40 +147,30 @@ func verifySessionCookie(ctx *middlewares.AutheliaCtx, targetURL *url.URL, userS
 	isUserAnonymous := userSession.Username == ""
 
 	if isUserAnonymous && userSession.AuthenticationLevel != authentication.NotAuthenticated {
-		return "", "", nil, nil, authentication.NotAuthenticated, fmt.Errorf("an anonymous user cannot be authenticated. That might be the sign of a compromise")
+		return "", "", nil, nil, authentication.NotAuthenticated, fmt.Errorf("an anonymous user cannot be authenticated (this might be the sign of a security compromise)")
 	}
 
-	if !userSession.KeepMeLoggedIn && !isUserAnonymous {
-		inactiveLongEnough, err := hasUserBeenInactiveTooLong(ctx)
-		if err != nil {
-			return "", "", nil, nil, authentication.NotAuthenticated, fmt.Errorf("unable to check if user has been inactive for a long time: %s", err)
+	if isSessionInactiveTooLong(ctx, userSession, isUserAnonymous) {
+		// Destroy the session a new one will be regenerated on next request.
+		if err = ctx.Providers.SessionProvider.DestroySession(ctx.RequestCtx); err != nil {
+			return "", "", nil, nil, authentication.NotAuthenticated, fmt.Errorf("unable to destroy session for user '%s' after the session has been inactive too long: %w", userSession.Username, err)
 		}
 
-		if inactiveLongEnough {
-			// Destroy the session a new one will be regenerated on next request.
-			err := ctx.Providers.SessionProvider.DestroySession(ctx.RequestCtx)
-			if err != nil {
-				return "", "", nil, nil, authentication.NotAuthenticated, fmt.Errorf("unable to destroy user session after long inactivity: %s", err)
-			}
-
-			ctx.Logger.Warnf("User %s has been inactive for too long", userSession.Username)
+		ctx.Logger.Warnf("Session destroyed for user '%s' after the session being inactive longer than the maximum timeout and the session not being marked as remembered", userSession.Username)
```

Comment:
```suggestion
		ctx.Logger.Warnf("Session destroyed for user '%s' after the session being inactive longer than the maximum timeout", userSession.Username)
```

Do we need to say anything about the session being remembered? Because we're assuming the remembered duration is long, it could actually still be sufficiently short enough that inactivity can still kick in.